license/usage-logs/README.md

```diff
@@ -0,0 +1,16 @@
+### Metered License Key Usage Logs
+When Using unioffice it is possible to see the usage logs of license key in every run. To enable this behavior all we have to do is setting the `UsageLog` to verbose and the log level to `Info` or above as follows.
```

Comment:
Probably also mention that this only available for api key based license(metered api key)